docs/index.adoc

```diff
@@ -4798,26 +4799,27 @@ edit [--open[=<editor>]] <file>
 One of the <<Optional Parameter Limitations, limitations of options with an optional parameter>> is that they are difficult to combine with positional parameters.
 
 With a custom parser plugin, we can customize the parser, such that `VALUE` in `--option=VALUE` is interpreted as the option parameter, and in `--option VALUE` (without the `=` separator), VALUE is interpreted as the positional parameter.
-The code below demonstrates:
+The code below demonstrates this:
 
 .Java
 [source,java,role="primary"]
 ----
 @Command(name = "edit")
 class Edit {
 
-    @Parameters(index = "0", description = "The file to edit.")
+    @Parameters(index = "0", arity="0..1", description = "The file to edit.")
     File file;
 
-   enum Editor { defaultEditor, eclipse, idea, netbeans }
+   enum Editor { eclipse, idea, netbeans }
 
    @Option(names = "--open", arity = "0..1", preprocessor = Edit.MyPreprocessor.class,
        description = {
-           "Optionally specify the editor to use; if omitted the default editor is used. ",
+           "Optionally specify the editor to use (${COMPLETION-CANDIDATES}). " +
+           "If omitted eclipse is used as default. ",
            "Example: edit --open=idea FILE opens IntelliJ IDEA (notice the '=' separator)",
-           "         edit --open FILE opens the specified file in the default editor"
+           "         edit --open FILE opens the specified file in eclipse"
        })
```

Comment:
Similar to the above, I think it is meaningful to have a `defaultEditor` choice, so let's keep the original description.